language-support/hs/examples/ex-haskell-grpc-arithmetic/ArithmeticServer.hs

```diff
@@ -0,0 +1,49 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE OverloadedLists #-}
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
Do we need all these extensions? I don't see any GADTs for instance. Same above.

Reply:
I checked. They are needed.